internal/pkg/config/root.go

```diff
@@ -29,10 +29,10 @@ type RootConf struct {
 	WWInternal      int           `yaml:"WW_INTERNAL"`
 	Comment         string        `yaml:"comment,omitempty"`
 	Ipaddr          string        `yaml:"ipaddr"`
-	Ipaddr6         string        `yaml:"ipaddr6,omitempty"`
+	Ipaddr6         string        `yaml:"Ipaddr6,omitempty"`
 	Netmask         string        `yaml:"netmask"`
```

Comment:
I would like to keep `ipaddr6` in the `nodes.conf` yaml file, as we have already `ipaddr` without capitalization.

Reply:
Unfortunately, when it's using lower case "ipaddr6", the script is not picking up the Ipaddr6 declaration from the warewulf.conf file. It's being skipped. I did not know where this was declared to make sure that the case is ignored.

Reply:
> the script

Which script? Or do you mean that a configured ipaddr6 value isn't actually available within Warewulf?

Reply:
The value ipaddr6 for the Warewulf controller node is not being picked up from the warewulf.conf

Reply:
I think we have the following situation, in `warewulf.conf` we use `ipaddr6`, but in all the templates (file ending with `.ww` it's `Ipaddr6` as this is an exported variable in golang which *must* be capitalized.
We have similar problems for the variables in `nodes.conf`, but we might need to sync the variable names in the long run.

Reply:
I updated the Ipaddr6 to use lower case. The ipaddr6 parameter in the /etc/warewulf/warewulf.conf should be with lower cased and all works fine:
e.g. 
ipaddr6: 2606:69c0:9120:786f:44:6468:6370:2
ipv6net: 2606:69c0:9120:786f::/64

Also we should be declaring an ipv6net instead of adding the network mask to the IPv6 address. When using IPv6 with network mask it breaks downloading the container/overlays files from ipxe via the IPv6 IP(/etc/warewulf/ipxe/default.ipxe):

```
set uri_base http://{{.Ipaddr6}}:{{.Port}}/provision/{{.Hwaddr}}?assetkey=${asset}&uuid=${uuid}
echo Warewulf Controller: {{.Ipaddr6}}
```

Reply:
@anderbubble we should merge this quite fast, as when testing this pr I realized that the actual ipv6 implementation is broken. This is also a bug fix.